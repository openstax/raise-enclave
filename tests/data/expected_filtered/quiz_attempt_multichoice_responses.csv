attempt_id,question_number,question_id,answer_id
6,1,574085b1-06f0-485b-9fe6-a784ba37e981,15
6,2,9368abf2-5a6c-4254-ab57-3edfdb338511,13
6,3,8a5008a7-970b-411b-9ed7-d1970b2bed56,9