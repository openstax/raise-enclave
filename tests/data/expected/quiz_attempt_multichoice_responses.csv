attempt_id,question_number,question_id,answer_id
0,1,574085b1-06f0-485b-9fe6-a784ba37e981,15
6,1,574085b1-06f0-485b-9fe6-a784ba37e981,15
0,2,9368abf2-5a6c-4254-ab57-3edfdb338511,13
6,2,9368abf2-5a6c-4254-ab57-3edfdb338511,13
0,3,8a5008a7-970b-411b-9ed7-d1970b2bed56,9
6,3,8a5008a7-970b-411b-9ed7-d1970b2bed56,9
1,1,6e6df72f-a6a3-424a-9ede-38b65720d2aa,32
1,2,da21613b-5d05-48d2-a07b-2bec166e5e54,29
1,4,2ffd03b0-67af-4e96-9476-5d1b621f1725,26
1,4,2ffd03b0-67af-4e96-9476-5d1b621f1725,25
1,5,0c6d9bee-7e89-4c87-8674-da0d56090b57,19
2,1,6e6df72f-a6a3-424a-9ede-38b65720d2aa,31
2,2,da21613b-5d05-48d2-a07b-2bec166e5e54,30
2,3,99134f44-0590-4fd3-86af-c926785b4a32,27
2,4,2ffd03b0-67af-4e96-9476-5d1b621f1725,26
2,5,0c6d9bee-7e89-4c87-8674-da0d56090b57,19
3,1,574085b1-06f0-485b-9fe6-a784ba37e981,15
3,2,9368abf2-5a6c-4254-ab57-3edfdb338511,13
3,3,8a5008a7-970b-411b-9ed7-d1970b2bed56,9
4,1,6e6df72f-a6a3-424a-9ede-38b65720d2aa,32
4,2,da21613b-5d05-48d2-a07b-2bec166e5e54,29
4,4,2ffd03b0-67af-4e96-9476-5d1b621f1725,26
4,4,2ffd03b0-67af-4e96-9476-5d1b621f1725,25
4,5,0c6d9bee-7e89-4c87-8674-da0d56090b57,19
5,1,6e6df72f-a6a3-424a-9ede-38b65720d2aa,32
5,2,da21613b-5d05-48d2-a07b-2bec166e5e54,29
5,4,2ffd03b0-67af-4e96-9476-5d1b621f1725,26
5,4,2ffd03b0-67af-4e96-9476-5d1b621f1725,25
5,5,0c6d9bee-7e89-4c87-8674-da0d56090b57,19